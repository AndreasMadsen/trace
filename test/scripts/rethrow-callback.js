function asyncFunction(callback) {
  process.nextTick(callback, new Error('error'));
}

function mainFunction() {
  asyncFunction(function nextTickFunction(err) {
    process.nextTick(() => {
      throw err;
    })
  });
}

mainFunction();
