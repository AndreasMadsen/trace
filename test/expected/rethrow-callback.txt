/scripts/rethrow-callback.js:8
      throw err;
      ^

Error: error
    at asyncFunction (/scripts/rethrow-callback.js:2:30)
    at mainFunction (/scripts/rethrow-callback.js:6:3)
    at Object.<anonymous> (/scripts/rethrow-callback.js:13:1)
    at ____________________:0:0
    at nextTickFunction (/scripts/rethrow-callback.js:7:13)
    at ____________________:0:0
    at asyncFunction (/scripts/rethrow-callback.js:2:11)
    at mainFunction (/scripts/rethrow-callback.js:6:3)
    at Object.<anonymous> (/scripts/rethrow-callback.js:13:1)
